File: archive/c/c/fizz-buzz.c
#include <stdio.h>

int main(void)
{
    for (unsigned int i = 1; i <= 100; i++) {
        if (i % 15 == 0) {
            puts("FizzBuzz");
        } else if (i % 3 == 0) {
            puts("Fizz");
        } else if (i % 5 == 0) {
            puts("Buzz");
        } else {
            printf("%u\n", i);
        }
    }
    return 0;
}

File: archive/r/ruby/fizz-buzz.rb
def fizzbuzz(n)
  result = +""

  result << "Fizz" if n % 3 == 0
  result << "Buzz" if n % 5 == 0

  puts(result.empty? ? n : result)
end

(1..100).each { fizzbuzz(it) }
